patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1354,12 +1354,25 @@
 +        return func_185467_w();
 +    }
 +
++    /**
++     * Gets the color that the beacon beam should be blended with.
++     *
++     * This should return null for no change, and a three entry float array with RGB values from 0.0 to 1.0.
++     *
++     * @param world The world the block is in
++     * @param blockState The state of the block
++     * @return The float array of colors to blend
++     * */
++    public float[] getBeaconColor(World world, IBlockState blockState) {
++        return null;
```

Comment:
On the one hand that float[] is exactly what mc then needs to process the color, on the other hand it provides very little type safety. I'm a bit torn here, should I recommend you use a better return value or not?